include/irr/builtin/materials/lambertian/singletexture/specializedshader_fragment

```diff
@@ -0,0 +1,13 @@
+#version 430 core
```

Comment:
instead of adding a `_fragment` suffix, do `.frag` its a universally recognized extension